README.md

```diff
@@ -54,43 +73,35 @@ LRAUV_keyboard_teleop
 
 > Tip: Type `LRAUV_` and press tab for autocomplete to show more example examples.
 
-## Using Docker
-
-You may also choose to use Docker for convenience. Make sure you have
-a recent version of [Docker](https://docs.docker.com/) and
-[nvidia-docker](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
-installed. Next to get started simply run the following command.
-```
-docker/build_and_run_docker.sh
-```
-
-To join in a separate terminal, remember to source Ignition and the workspace:
-```
-docker/join.sh mbari_lrauv
-. /home/ign_ws/install/setup.bash
-. /home/colcon_ws/install/setup.bash
-```
-
 ## To test integration with MBARI LRAUV code base
 
-MBARI's code base lives in a private repository. For people with access,
+MBARI's code base lives in a separate, private repository.
+For people with access,
 [here](https://bitbucket.org/mbari/lrauv-application/src/7b3b5fce1b0ad1af1734952adaf94f2a69193aec/docker_ignition/?at=feature%2F2021-02-12-ignition-sim)
-are instructions on setting it up from source.
+are instructions on setting it up from source and compiling.
+
+Alternatively, you can use the public Docker image (see below).
 
 The integration assumes that this repository is cloned as a sibling of
 the `lrauv-application` repository, i.e.:
 ```
 <workspace>
 |-- lrauv
-`-- lrauv-application
+|-- lrauv-application
```

Comment:
Nitpick

```suggestion
└-- lrauv-application
```

Reply:
Tools like `tree` use \` instead of `└` if the latter character is not supported. That's the case for most of my docker containers :cry: